fabtests/functional/inj_complete.c

```diff
@@ -146,7 +155,7 @@ static int run_test(void)
 				return ret;
 		}
 	}
-	fprintf(stdout, "GOOD: Completed FI_INJECT_COMPLETIONS Testing\n");
+	fprintf(stdout, "GOOD: Completed %s Testing\n", fi_tostr(&flag, FI_TYPE_OP_FLAGS));
```

Comment:
Please check line lengths here and above and below, and limit to around 80 characters.

Reply:
Thanks for pointing it out. Addressed in latest changes.